diff --git a/testdata/test_with_diff.go b/testdata/test_with_diff.go
index 089369d..4ea2f97 100644
--- a/testdata/test_with_diff.go
+++ b/testdata/test_with_diff.go
@@ -5,3 +5,5 @@ func a(string, string, string, string, string, string, string, string, string, s
 }
 
 var A = a(`123`, `123`, `123`, `123`, `123`, `123`, `123`, `123`, `123`, `123`)
+
+var B = a(`123`, `123`, `123`, `123`, `123`, `123`, `123`, `123`, `123`, `123`)
